### ISSUE TITLE
Close and Open Project Expands other Collapsed Projects

### ISSUE THREAD

**Issue by joewheaton** [Riverscapes/QRAVEPlugin#32]
Version: 0.5.1

Part of keeping the QRAVE tree from being over-whelming is controlling (with businesses logic collapsed tag) whether a project tree is collapsed or not. When a user takes time to control the collapsed state of other open projects, we should NOT mess with that. What is happening is when you open a new project, or close an existing project, then the other projects loose their collapsed state and expand out. It makes it really hard to see the project you just opened, and will annoy users. See video:

[![Alt-text](https://img.youtube.com/vi/F9RwWEpWRFc/0.jpg)](https://www.youtu.be/watch?v=F9RwWEpWRFc)

**Comment by MattReimer** [comment]
Agreed. Not sure if we need to track expand/collapse state of every item but there may be some middle ground here.
